### PR TITLE
fix doc of erf,rank,mm,cross_entropy,pixel_shuffle,kron...

### PR DESCRIPTION
**Pull request by xmy0916**
### PR types

Bug fixes 
### PR changes

Docs
### Describe

fix doc of erf,rank,mm,cross_entropy,pixel_shuffle,kron...

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -11297,25 +11297,24 @@ def shape(input):
...
 
-            import paddle.fluid as fluid
+            import paddle
 
-            input = fluid.data(name="input", shape=[3, 100, 100], dtype="float32")
-            rank = fluid.layers.rank(input) # rank=(3,)
+            input = paddle.rand((3, 100, 100))
+            rank = paddle.rank(input)
+            print(rank)
+            # Tensor(shape=[], dtype=int32, place=CUDAPlace(0), stop_gradient=True,
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py]: 这里直接注释 # 3 就好了 不用写其他的信息

File: python/paddle/fluid/layers/ops.py [hunk 2]
```diff
@@ -300,12 +300,12 @@
...
 
         import paddle
-        paddle.disable_static()
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = paddle.cosh(x)
-        print(out.numpy())
-        # [1.08107237 1.02006676 1.00500417 1.04533851]
+        print(out)
+        # Tensor(shape=[4], dtype=float32, place=CPUPlace, stop_gradient=True,
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 这里也是 直接  [1.08107237 1.02006676 1.00500417 1.04533851] 就好 其他地方同理 就不一一写了

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -11297,25 +11297,22 @@ def shape(input):
...
     Examples:
         .. code-block:: python
 
-            import paddle.fluid as fluid
+            import paddle
 
-            input = fluid.data(name="input", shape=[3, 100, 100], dtype="float32")
-            rank = fluid.layers.rank(input) # rank=(3,)
+            input = paddle.rand((3, 100, 100))
+            rank = paddle.rank(input)
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py:11297]: 这里需要补充输出信息

File: python/paddle/fluid/layers/ops.py [hunk 4]
```diff
@@ -95,7 +95,6 @@
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = F.sigmoid(x)
-        print(out.numpy())
         # [0.40131234 0.450166   0.52497919 0.57444252]
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 这里需要加上 print(out)

File: python/paddle/fluid/layers/ops.py [hunk 5]
```diff
@@ -110,7 +109,6 @@
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = F.log_sigmoid(x)
-        print(out.numpy())
         # [-0.91301525 -0.79813887 -0.64439666 -0.55435524]
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 同上

File: python/paddle/fluid/layers/ops.py [hunk 6]
```diff
@@ -124,7 +122,6 @@
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = paddle.exp(x)
-        print(out.numpy())
         # [0.67032005 0.81873075 1.10517092 1.34985881]
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 同上

File: python/paddle/fluid/layers/ops.py [hunk 7]
```diff
@@ -138,7 +135,6 @@
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = paddle.tanh(x)
-        print(out.numpy())
         # [-0.37994896 -0.19737532  0.09966799  0.29131261]
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 同上

File: python/paddle/fluid/layers/ops.py [hunk 8]
```diff
@@ -152,7 +148,6 @@
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = paddle.atan(x)
-        print(out.numpy())
         # [-0.38050638 -0.19739556  0.09966865  0.29145679]
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 同上

File: python/paddle/fluid/layers/ops.py [hunk 9]
```diff
@@ -180,7 +175,6 @@
 
         x = paddle.to_tensor([0.1, 0.2, 0.3, 0.4])
         out = paddle.sqrt(x)
-        print(out.numpy())
         # [0.31622777 0.4472136  0.54772256 0.63245553]
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 同上

File: python/paddle/fluid/layers/ops.py [hunk 10]
```diff
@@ -206,7 +200,6 @@
 
         x = paddle.to_tensor([-0.4, -0.2, 0.1, 0.3])
         out = paddle.abs(x)
-        print(out.numpy())
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/ops.py]: 同上

File: python/paddle/nn/functional/loss.py [hunk 11]
```diff
@@ -1165,23 +1165,20 @@ def cross_entropy(input,
...
             import paddle
-            paddle.disable_static()
             input_data = np.random.random([5, 100]).astype("float64")
             label_data = np.random.randint(0, 100, size=(5)).astype(np.int64)
             weight_data = np.random.random([100]).astype("float64")
             input =  paddle.to_tensor(input_data)
             label =  paddle.to_tensor(label_data)
             weight = paddle.to_tensor(weight_data)
             loss = paddle.nn.functional.cross_entropy(input=input, label=label, weight=weight)
-            print(loss.numpy())
```
Comment (TCChenlong) [inline python/paddle/nn/functional/loss.py]: 同上

File: python/paddle/nn/functional/vision.py [hunk 12]
```diff
@@ -347,11 +343,9 @@ def pixel_shuffle(x, upscale_factor, data_format="NCHW", name=None):
             import paddle.nn.functional as F
             import numpy as np
             x = np.random.randn(2, 9, 4, 4).astype(np.float32)
-            paddle.disable_static()
             x_var = paddle.to_tensor(x)
             out_var = F.pixel_shuffle(x_var, 3)
             out = out_var.numpy()
-            print(out.shape)
```
Comment (TCChenlong) [inline python/paddle/nn/functional/vision.py:352]: 少了print(out.shape) 与 结果

File: python/paddle/nn/layer/loss.py [hunk 13]
```diff
@@ -186,9 +183,8 @@ class CrossEntropyLoss(fluid.dygraph.Layer):
             and does not contribute to the input gradient. Default is ``-100``.
 
     Returns:
-        The tensor variable storing the cross_entropy_loss of input and label.
+        Tensor. The tensor storing the cross_entropy_loss of input and label.
 
-    Return type: Variable.
```
Comment (TCChenlong) [inline python/paddle/nn/layer/loss.py:207]: 这里的代码需要更新一下 和中文的示例代码保持一致

File: python/paddle/nn/layer/loss.py [hunk 14]
```diff
@@ -186,44 +183,20 @@ class CrossEntropyLoss(fluid.dygraph.Layer):
...
-            import paddle.fluid as fluid
-            import numpy as np
-
-            input = fluid.data(name='input', shape=[5, 100], dtype='float64')
-            label = fluid.data(name='label', shape=[5], dtype='int64')
-            weight = fluid.data(name='weight', shape=[100], dtype='float64')
+            input_data = paddle.uniform([5, 100],dtype = "float64")
+            label_data = paddle.uniform(0, 100, size=(5),dtype = "int64")
+            weight_data = paddle.uniform([100],dtype = "float64")
+            input  = paddle.to_tensor(input_data)
```
Comment (TCChenlong) [inline python/paddle/nn/layer/loss.py]: 195~197需要删除 用 paddle.uniform 创建的就是 tensor ，就不需要to_tensor 了

File: python/paddle/tensor/math.py [hunk 15]
```diff
@@ -815,42 +812,28 @@ def mm(input, mat2, name=None):
...
-
             import paddle
-            import paddle.fluid as fluid
-            x = fluid.data(name='x', shape=[2, 3], dtype='float32')
-            mat2 = fluid.data(name='mat2', shape=[3, 2], dtype='float32')
-            out = paddle.mm(x, mat2) # out shape is [2, 2]
+            input = paddle.arange(1, 7).reshape((3, 2)).astype('float32')
+            mat2 = paddle.arange(1, 9).reshape((2, 4)).astype('float32')
+            out = paddle.mm(input, mat2)
+            # Tensor(shape=[3, 4], dtype=float32, place=CPUPlace, stop_gradient=True,
```
Comment (TCChenlong) [inline python/paddle/tensor/math.py]: 这里需要改一下输出

File: python/paddle/tensor/math.py [hunk 16]
```diff
@@ -1416,13 +1399,11 @@ def log10(x, name=None):
 
 def addcmul(input, tensor1, tensor2, value=1.0, name=None):
```
Comment (TCChenlong) [inline python/paddle/tensor/math.py]: 这个API应该是要删除？但是没有删？建议和mentor确认下

File: python/paddle/tensor/math.py [hunk 17]
```diff
@@ -1638,49 +1620,36 @@ def __check_input(input, offset, dim1, dim2):
...
-          #[[ 1.  2.  3.  2.  4.  6.]
-          # [ 4.  5.  6.  8. 10. 12.]
-          # [ 7.  8.  9. 14. 16. 18.]
-          # [ 3.  6.  9.  4.  8. 12.]
-          # [12. 15. 18. 16. 20. 24.]
-          # [21. 24. 27. 28. 32. 36.]]
+            x = paddle.to_tensor([[1, 2], [3, 4]], dtype='int64')
+            y = paddle.to_tensor([[1, 2, 3], [4, 5, 6], [7, 8, 9]], dtype='int64')
+            out = paddle.kron(x, y)
+            # Tensor(shape=[6, 6], dtype=int64, place=CPUPlace, stop_gradient=True,
```
Comment (TCChenlong) [inline python/paddle/tensor/math.py]: print(out)
然后改一下输出的格式

File: python/paddle/nn/layer/loss.py [hunk 18]
```diff
@@ -186,44 +183,20 @@ class CrossEntropyLoss(fluid.dygraph.Layer):
...
+            label  = paddle.to_tensor(label_data)
+            weight = paddle.to_tensor(weight_data)
             ce_loss = paddle.nn.loss.CrossEntropyLoss(weight=weight, reduction='mean')
```
Comment (TCChenlong) [inline python/paddle/nn/layer/loss.py]: paddle.nn.loss.CrossEntropyLoss -> paddle.nn.CrossEntropyLoss

File: python/paddle/nn/functional/loss.py (hunk 11 above)
Comment (TCChenlong) [inline python/paddle/nn/functional/loss.py]: 少了 print(loss) 以及结果

File: python/paddle/nn/layer/loss.py [hunk 19]
```diff
@@ -186,44 +183,20 @@ class CrossEntropyLoss(fluid.dygraph.Layer):
...
             output = ce_loss(input, label)
```
Comment (TCChenlong) [inline python/paddle/nn/layer/loss.py:220]: 少了print(output)与结果

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM
